tensorflow/python/training/input.py

```diff
@@ -722,10 +722,8 @@ def shuffle_batch(tensor_list, batch_size, capacity, min_after_dequeue,
         dtypes=types, shapes=shapes, shared_name=shared_name)
     _enqueue(queue, tensor_list, num_threads, enqueue_many)
     full = (math_ops.cast(math_ops.maximum(0, queue.size() - min_after_dequeue),
-                          dtypes.float32) *
-            math_ops.truediv(ops.convert_to_tensor(1,dtype=dtypes.float32) ,
-                ops.convert_to_tensor(capacity -
-                    min_after_dequeue,dtype=dtypes.float32)))
+        dtypes.float32) *
+        (constant_op.constant(1.) / (capacity - min_after_dequeue)))
```

Comment:
Apologies: I should have noticed this before.  The code is `x * 1 / y`, but should be simplified to `x / y`.  Also, please fix the formatting to

    full = (math_ops.cast(math_ops.maximum(0, queue.size() - min_after_dequeue),
                          dtypes.float32) / (capacity - min_after_dequeue))